DeepMRSeg/deepmrseg_loadmodel.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python3
```

Comment:
This creates an executable but there is no help. I get the following error:

Singularity> deepmrseg_downloadmodel
Traceback (most recent call last):
  File "/usr/local/bin/deepmrseg_downloadmodel", line 33, in <module>
    sys.exit(load_entry_point('DeepMRSeg==0.1.0', 'console_scripts', 'deepmrseg_downloadmodel')())
  File "/usr/local/bin/deepmrseg_downloadmodel", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/local/lib/python3.6/dist-packages/importlib_metadata/__init__.py", line 96, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'DeepMRSeg.deepmrseg_downloadmodel'

Reply:
Fixed